### PR TITLE
DrivAerML: T_max=34 + lr=4.8e-4 MSE-only (schedule fine-tuning below T_max=36)

### PR DESCRIPTION
**Pull request by tcapelle**
## Hypothesis

T_max=36 was the clear schedule winner at lr=5e-4 (val=3.622% vs T_max=30=3.833%, T_max=40=3.651%). But at lr=4.8e-4, the optimal schedule may shift slightly. T_max=34 probes the sub-optimum just below 36.

## Instructions

Run DrivAerML at **T_max=34** with lr=4.8e-4:

```bash
cd target/icml2026 && SENPAI_TIMEOUT_MINUTES=600 python train.py \
  --dataset drivaerml \
  --optimizer adamw \
  --lr 4.8e-4 \
  --cosine-t-max 34 \
  --grad-clip 0.5 \
  --enable-fourier \
  --model-layers 4 --model-hidden-dim 512 --model-heads 8 \
  --epochs 999 --batch-size 1 \
  --drivaerml-train-surface-points 50000 \
  --drivaerml-eval-surface-points 50000 \
  --max-train-batches 394 --max-eval-batches 200 \
  --ema-decay 0.9995 --ema-mode fixed \
  --no-compile-model --save-checkpoint \
  --wandb-group dm-lr48e4-tmax36 \
  --agent levi \
  --wandb-name "levi/dm-lr48e4-tmax34"
```

Key changes: `--lr 4.8e-4 --cosine-t-max 34` (MSE-only). If checkpoint saved, run full-eval with `--max-eval-batches 0 --load-checkpoint <path> --eval-only`.

## Baseline

- **DM best val:** 3.622% (T_max=36 MSE-only at lr=5e-4)
- **DM best TEST:** 4.117% (gojo lr=4.8e-4 T_max=30, PR #3308)
- **T_max=40+lr=4.8e-4 (alphonse #3388):** running now — T_max=34 brackets from below
- **Target: val < 3.622%**
- **Reproduce:** `cd target/icml2026 && SENPAI_TIMEOUT_MINUTES=600 python train.py --dataset drivaerml --optimizer adamw --lr 5e-4 --cosine-t-max 36 --grad-clip 0.5 --enable-fourier --model-layers 4 --model-hidden-dim 512 --model-heads 8 --epochs 999 --batch-size 1 --drivaerml-train-surface-points 50000 --drivaerml-eval-surface-points 50000 --max-train-batches 394 --max-eval-batches 200 --ema-decay 0.9995 --ema-mode fixed --no-compile-model --save-checkpoint`

### CODE REVIEW

**Comment by morganmcg1** [comment]
HUMAN: Closing during the final cost-control harvest pass. The associated student pod (levi) was intentionally stopped after its .claude logs were archived locally, and this PR was not in the small set kept running for final test-metric harvesting. Reason: the pod was burning time on an old debug command rather than a valid final-harvest training run.
